src/simularium/VisData.ts

```diff
@@ -376,15 +381,29 @@ class VisData {
         }
     }
 
+    public setCacheDisabled(cacheDisabled: boolean): void {
+        this.disableCache = cacheDisabled;
+    }
+
     // Add parsed frames to the cache and save the timestamp of the first frame
     private addFramesToCache(frames: ParsedBundle): void {
+        if (this.disableCache) {
+            this.trimCacheHead();
+        }
         Array.prototype.push.apply(this.frameDataCache, frames.frameDataArray);
         Array.prototype.push.apply(
             this.frameCache,
             frames.parsedAgentDataArray
         );
     }
 
+    private trimCacheHead(): void {
+        while (this.frameCache.length > 1) {
+            this.frameCache.shift();
+            this.frameDataCache.shift();
```

Comment:
is it possible to just not store the cache instead of storing it and then deleting it?

Reply:
I tried my suggestions and it doesn't render anything: I guess I don't fully understand how this cache is currently working